specification/servicefabricmanagedclusters/resource-manager/readme.md

```diff
@@ -201,6 +212,17 @@ input-file:
 - Microsoft.ServiceFabric/preview/2021-01-01-preview/nodetype.json
 ```
 
+### AutoRest v3 Suppressions
+``` yaml
+suppressions:
+    
+  - code: ResourceNameRestriction
+    reason: the service does not have a patter restriction. ARM's default rule should get applied.
```

Comment:
>  reason: the service does not have a patter restriction. ARM's default rule should get applied. [](http://example.com/codeflow?start=2&length=96)

Adding a pattern to document ARM's default rule should be an acceptable option here.

Reply:
I couldn't find what is the pattern accepted by ARM. we don't have any specific restriction so ARM should do the validation